### PR TITLE
Canny Actions Destination

### PR DESCRIPTION
**Pull request by stayseesong**

### CODE REVIEW

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+4. Search fpr _Canny (Actions)_ and select it.
+5. Click **Configure Canny (Actions)**.
+6. Select an existing Source to connect to Canny (Actions).
+7. Enter your configuration settings.
+8. Update the API key in the Canny (Actions) destination settings.
+9. Enable the destination and save changes.
+
+{% include components/actions-fields.html %}
+
 ---
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/canny-functions/index.md:39]:
```suggestion

```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+
+## Benefits of Canny (Actions) vs Canny Classic
+
+Canny (Actions) provides the following benefit over the classic Canny destination:
+- **Group Events**. Canny (Actions) supports group events and updates or adds properties to companies.
+
+## Getting started
+
+1. Go to your [Canny Admin Segment Settings](https://canny.io/redirect?to=%2Fadmin%2Fsettings%2Fsegment){:target="\_blank"}.
+2. Obtain your API key by installing the integration.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/canny-functions/index.md:28]:
```suggestion
2. Install the integration to get your API key.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+3. From the Segment web app, navigate to **Connections > Catalog**, then select the **Destinations** tab in the catalog.
+4. Search fpr _Canny (Actions)_ and select it.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/canny-functions/index.md:30]:
```suggestion
4. Search for **Canny (Actions)** and select it.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+5. Click **Configure Canny (Actions)**.
+6. Select an existing Source to connect to Canny (Actions).
+7. Enter your configuration settings.
+8. Update the API key in the Canny (Actions) destination settings.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/canny-functions/index.md:34]:
```suggestion
8. Enter the Canny API key in the Canny (Actions) destination settings.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by tcgilbert — APPROVED** [review]

**Comment by BR4VE** [comment]
@tcgilbert @stayseesong What is the ETA for this? We'd like to ship our integration soon.

**Pull request merged**